mock-core-configs/etc/mock/amazonlinux-2023-aarch64.cfg

```diff
@@ -0,0 +1,5 @@
+include('templates/amazonlinux-2023.tpl')
+
+config_opts['root'] = 'amzn2023'
```

Comment:
Please change this to `amazonlinux-2023-aarch64`

Reply:
I had even half done this - it's in the template as `amazonrinux-2023-{{ target_arch }}` already, and I'd just forgotten to delete this line. Fixing.